src/panels/lovelace/common/directives/long-press-directive.ts

```diff
@@ -1,8 +1,5 @@
 import { directive, PropertyPart } from "lit-html";
-// See https://github.com/home-assistant/home-assistant-polymer/pull/2457
```

Comment:
I feel like this could have been done in another PR . Fine for now but let's keep PRs simpler in the future. This PR is now 800 lines long ;)